tests/Earthfile

```
@@ -1413,6 +1414,13 @@ test-init-golang:
     DO +RUN_EARTHLY --target "+build"
     DO +RUN_EARTHLY --target "+test"
 
+test-reserved-label:
+   DO +RUN_EARTHLY \
+       --earthfile=reserved-label.earth \
+       --target=+test1 \
+       --should_fail=true \
+       --output_contains="forbidden"
```

Is it possible to expand this to something more complete? 

This field is actually a regex. The quotes in the error message might make it difficult to include, but you can just use a `.` in place of the `"` to simplify the pattern.


```suggestion
       --output_contains="LABEL keys starting with .dev.earthly.. are forbidden"
```